pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+# Description
+
+<!--- Please include a summary of the changes and the related issue. -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Type of change
```

Comment:
Maybe we should a subsection here about adding Issues if the corresponds with any issue.

Reply:
Good point